### PR TITLE
Convert copied articles to ES6

### PR DESCRIPTION
**Pull request by piuccio**
and write a test for it

### CODE REVIEW

File: facia-tool/public/js/models/config/persistence.js [hunk 1]
```diff
@@ -1,114 +1,137 @@
...
+        return false;
+    }
+}
+
+function postUpdate (opts) {
+    return publicInterface[postUpdateSym](opts);
+}
+
+function createFrontFromCollection (collection) {
+    var front = collection.parents()[0],
```
Comment (uplne) [inline facia-tool/public/js/models/config/persistence.js]: Why not to use const?

File: facia-tool/public/js/models/group.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ define([
     mediator
 ) {
     mediator = mediator.default;
+    copiedArticle = copiedArticle.default;
```
Comment (uplne) [inline facia-tool/public/js/models/group.js]: why some modules are using import and some not?

File: facia-tool/public/js/models/config/persistence.js (hunk 1 above)
Comment (piuccio) [inline facia-tool/public/js/models/config/persistence.js]: i think `const` is misleading and useless. It's a `const` reference, not a `const` value, people reading `const front` might expect that `front` doesn't change, but it does.

File: facia-tool/public/js/models/group.js (hunk 2 above)
Comment (piuccio) [inline facia-tool/public/js/models/group.js]: I'm in the process of rewriting all of them, but I'm doing in steps... It takes time

**Comment by uplne** [comment]
:+1:

**Pull request merged**